utils/generic-components.stanza

```diff
@@ -928,7 +928,7 @@ public pcb-module test-points (count:Int, method:ocdb/utils/generic-components/T
 
 public defn add-testpoint (rs:Tuple<JITXObject>, method:ocdb/utils/generic-components/Testpoint|Instantiable) :
   inside pcb-module :
-    val unique-id = symbol-join([`TP- to-int(hash(rs))])
+    val unique-id = symbol-join([`TP- to-int(hash(map(local, rs)))])
```

Comment:
@tjknoth Is there an easier way in the new object core to create a non-colliding id for the `make-inst()` function?

Reply:
Can you just use `genid()`?
There is also `index(local)` which is defined on object-core/Object -- but it is probably not exposed to users. We can do so if need be.